test/integration/query.spec.ts

```diff
@@ -64,8 +63,9 @@ describe("Test Query Metrics", function() {
 
       while (queryIterator.hasMoreResults()) {
         const { resources: results, queryMetrics, activityId, requestCharge } = await queryIterator.fetchNext();
-        assert(activityId, "activityId must exist");
-        assert(requestCharge, "requestCharge must exist");
+        // TODO: stfaul - plumb activityId and requestCharge through the iterator and renable these
```

Comment:
@christopheranderson We need to land your other PR or I need to pipe these values through

Reply:
https://github.com/Azure/azure-cosmos-js/issues/301

Reply:
This works now that we attempt to hit the gateway with the query first.